azure-devops/azext_devops/dev/team/arguments.py

```diff
@@ -22,27 +21,24 @@
                                           SERVICE_ENDPOINT_AUTHORIZATION_SERVICE_PRINCIPAL]
 
 
+# pylint: disable=line-too-long
```

Comment:
This one can be fixed. Line length limit is to improve readability. 
We should not disable all pylint warnings to fix CI :)

Reply:
agree, but it's hard to validate style changes locally right now